answer_files/2008_r2_core/Autounattend.xml

```diff
@@ -49,8 +49,8 @@
                     <InstallToAvailablePartition>false</InstallToAvailablePartition>
                     <InstallFrom>
                         <MetaData wcm:action="add">
-                            <Key>/IMAGE/NAME</Key>
-                            <Value>Windows Server 2008 R2 SERVERSTANDARDCORE</Value>
+                            <Key>/image/index</Key>
+                            <Value>1</Value>
```

Comment:
I've ditched that patch from the patchset - it makes no sense to have half the references as /IMAGE/NAME and half as /image/index.  It makes it a bit more fiddly for people using alternative ISOs but I guess that's too bad.